features/FxA-72-reset-password/README.md

```diff
@@ -0,0 +1,28 @@
+
+Reset Password
+==============
+
+https://mozilla.aha.io/features/FXA-72
+
+So that I can avoid inadvertently losing all my Sync data, as a forgetful Firefox Accounts user, I want to understand more clearly what will happen when I reset my password.
+
+## Current/Proposed UX
+![firefox-sync](https://www.lucidchart.com/publicSegments/view/4d816217-8c5a-48be-a14e-79d8eac303d8/image.png)
```

Comment:
@ryanfeeley showed this to me yesterday and we tweaked the language a bit so it doesn't say "unsaved sync data" anymore. I feel that linking to external assets in these feature issues somewhat defeats their purpose. If the externally linked assets change, how do the relevant parties learn and acknowledge the change? I felt that the intention was that these feature definitions were self-contained and any changes to them required a PR.